assets/airports/eddf.json

```diff
@@ -450,11 +528,11 @@
       "entryPoints": {
         "T173": ["GAPLA", ["DEMAB", "A160-"], ["SOLVU", "A130-"]]
```

Comment:
`T173` is not a fix, which is what the game expects it to be. It is an airway, but it is not defined as an airway either, so the game really doesn't know what to make of it. When it is told there is an "entry point" of "T173", it tells the plane to fly to "T173" and then proceed along the `KERAX25N` star until you reach `DF421`.

So this should be amended to the first point, where the aircraft is to join the STAR route, which would be `GAPLA` in this case.
```javascript
"entryPoints": {
  "GAPLA": ["GAPLA", ["DEMAB", "A160-"], ["SOLVU", "A130-"]]
},
```

This will need to be corrected throughout the STARs in order for them to be interpreted correctly by the sim.